src/Symfony/Component/Uid/Tests/UlidTest.php

```diff
@@ -237,4 +238,22 @@ public function testFromStringOnExtendedClassReturnsStatic()
     {
         $this->assertInstanceOf(CustomUlid::class, CustomUlid::fromString((new CustomUlid())->toBinary()));
     }
+
+    /**
+     * @testWith    ["00000000-0000-0000-0000-000000000000"]
```

Comment:
`@dataProvider` for consistency?

Reply:
I very much prefer `@testWith` when possible personnaly.